### PR TITLE
210309 spring oobrelease gzh

### PR DESCRIPTION
**Pull request by backwind1233**
update release notes for Spring February OOB release

### CODE REVIEW

File: releases/2021-02/java.md [hunk 1]
```diff
@@ -391,7 +395,136 @@ To use the GA and beta libraries, refer to the Maven dependency information belo
...
+  <artifactId>azure-spring-cloud-starter-eventhubs-kafka</artifactId>
+  <version>2.2.0</version>
+</dependency>
+
+<dependency>
+  <groupId>com.azure.spring</groupId>
+  <artifactId>azure-spring-cloud-stream-binder-eventhubs</artifactId>
+  <version>2.2.0</version>
+</dependency>
+```
```
Comment (yiliuTo) [inline releases/2021-02/java.md:436]: I think we could add `azure-spring-cloud-starter-storage-queue` here as well.

**Review by saragluna — APPROVED** [review]
LGTM.

**Review by yiliuTo — APPROVED** [review]
LGTM

File: releases/2021-02/java.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ The Azure SDK team is pleased to announce our February 2021 client library relea
 - Azure Cosmos
 - Azure Search Documents
 - Azure Messaging Event Hubs
+- Azure Spring Boot
```
Comment (weshaggard) [inline releases/2021-02/java.md:18]: Did you want to add these to the February release notes or the March?

**Pull request merged**